package.json

```diff
@@ -19,6 +19,7 @@
     "body-parser": "~1.17.1",
     "bookshelf": "^0.10.4",
     "bookshelf-bcrypt": "^2.1.0",
+    "chai": "^4.1.0",
```

Comment:
what was the purpose of moving this into production dependencies? just to get the linter to shut up?